ostd/src/sync/rcu/mod.rs

```diff
@@ -232,7 +234,7 @@ impl<P: NonNullPtr + Send> RcuReadGuardInner<'_, P> {
         }
 
         if let Some(p) = NonNull::new(self.obj_ptr) {
-            // SAFETY: It was previously returned by `into_raw`.
+            // SAFETY: The pointer was previously returned by `into_raw`.
```

Comment:
I originally noticed that the safety comments here were very imprecise, so I intended to submit another PR to revise them.

But it seems that you're interested in touching those lines as well. If so, I think we'd better just improve them: https://github.com/lrh2000/asterinas/commit/6856574a5faf5a35cdd8d386d71902d2e9d6a885